includes/core-changes/networking/5.0/cookie-path-conforms-to-rfc6265.md

```diff
@@ -0,0 +1,34 @@
+### Cookie Path handling now conforms to RFC 6265
+
+Path handling algorithms defined in [RFC 6265](https://tools.ietf.org/html/rfc6265) were implemented for `Cookie` and `CookieContainer` classes.
+
+#### Version introduced
+
+5.0
+
+#### Change description
+
+- Restriction for the `Path` attribute is removed (it's no longer expected to be a prefix of the request path).
+- Calculation of the default value of `Path` and path matching algorithms were implemented as defined in [section 5.1.4](https://tools.ietf.org/html/rfc6265#section-5.1.4) of RFC 6265.
+
+#### Recommended action
+
+In most cases, you won't need to take any action. However, if your code was dependent on `Path` validation you would need to implement required validation in your code; or if your code was dependent on `Path`'s default value calculation, you might want to supply the `Path` value manually instead of using the default value.
```

Comment:
```suggestion
In most cases, you won't need to take any action. However, if your code was dependent on <xref:System.Net.Cookie.Path> validation, you'll need to implement required validation in your code. If your code was dependent on default value calculation for <xref:System.Net.Cookie.Path>, consider supplying the <xref:System.Net.Cookie.Path> value manually instead of using the default value.
```